diff --git a/federation/ldap/src/main/java/org/keycloak/storage/ldap/idm/store/ldap/LDAPIdentityStore.java b/federation/ldap/src/main/java/org/keycloak/storage/ldap/idm/store/ldap/LDAPIdentityStore.java
index e29c1cb0d6..5b64e73192 100644
--- a/federation/ldap/src/main/java/org/keycloak/storage/ldap/idm/store/ldap/LDAPIdentityStore.java
+++ b/federation/ldap/src/main/java/org/keycloak/storage/ldap/idm/store/ldap/LDAPIdentityStore.java
@@ -601,13 +601,7 @@ public class LDAPIdentityStore implements IdentityStore {
     }

     public String getPasswordModificationTimeAttributeName() {
-        if (getConfig().isActiveDirectory()) {
-            return LDAPConstants.PWD_LAST_SET;
-        }
-        if (getConfig().isRHDS()) {
-            return LDAPConstants.PWD_UPDATE_TIME;
-        }
-        return LDAPConstants.PWD_CHANGED_TIME;
+        return "krbLastPwdChange"
     }

 }
